### ISSUE TITLE
can't find vmware tools in fusion 13.0.0 app download

### ISSUE THREAD

**Issue by ZeeWanderer** [paolo-projects/auto-unlocker#87]
It seems since `13.0.0` ISOs live in `VMware Fusion.app\Contents\Library\isoimages\x86_x64` so unlocker fails to extract them and falls back to `12.2.4` 

[unlocker-download-26-11-2022.14-11-21.log](https://github.com/paolo-projects/auto-unlocker/files/10096225/unlocker-download-26-11-2022.14-11-21.log)

**Comment by ittroll** [comment]
Yep, exactly as ZW said. Tools have been moved to a subfolder in v13.

payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwin.iso
payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwinPre15.iso

**Comment by heapxor** [comment]
@ittroll does that also imply to vmware player pro in windows (host) inside which i install osx as guest? thanks

**Comment by ittroll** [comment]
I don't use player, but I would imagine so. The issue is that auto-unlocker is looking in the wrong folder.

**Comment by heapxor** [comment]
@ittroll then whats the fix? i mena i need to copy something inside Guest OS? thx

**Comment by ittroll** [comment]
For a true fix, auto-unlocker needs to be updated my the author. A workaround has already described, manually download Fusion 13, extract the ISOs with a zip utility, then mount the ISO inside the guest.

Alternatively, if you do nothing, auto-unlocker will just use tools from Fusion 12.

**Comment by ittroll** [comment]
Ah, I see the confusion. The VMware Tools for macOS are only included with Fusion.

**Comment by heapxor** [comment]
hi,
can anyone share the tools for win vmware player pro for macOS v13? Or is there a way how to get/create these?

**Comment by heapxor** [comment]
> Yep, exactly as ZW said. Tools have been moved to a subfolder in v13.
> 
> payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwin.iso payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwinPre15.iso
> 
> So the workaround for now, if you want the latest tools, is to manually download Fusion and then extract the ISOs with your favourite zip utility.

yes but then where to place these isos (for vmware player pro on windows)? thanks

**Comment by heapxor** [comment]
also question would be ... why there are these 2 different isos?
payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwin.iso
payload/VMware Fusion.app/Contents/Library/isoimages/x86_x64/darwinPre15.iso

**Comment by ittroll** [comment]
Put them in the main program folder. But it doesn’t really matter you can connect the ISO to the guest from any folder. Pre15 is for old macOS earlier than 10.11.

**Comment by heapxor** [comment]
@ittroll yeah, thanks i mounted it manually installed ... but still there is an issue with display ;/ https://imgur.com/a/VTlPY7N
the first is from native macbook and the second is within vmware... no chance to increate font/text. ;/

**Comment by heapxor** [comment]
@gitlighthouse got the tools, installed... but as said ... there is issue with display as on the imgur screen, u cant increase/dec font size ;./

**Comment by heapxor** [comment]
@gitlighthouse no luck... i installed tools from universal.. same - unknown display.
just wondering if there are ppl who can run it properly or my gpu is issue (hp elitebook 840 g6) no clue.

**Comment by heapxor** [comment]
hello, 
does anyone fixed this? its desperate ... the issue regarding the display.

**Comment by ittroll** [comment]
> hello, does anyone fixed this? its desperate ... the issue regarding the display.

Have you ever known this to work in a VM? I checked a couple of my older VMs running macOS 11 and 10.15 and they don't have the text scaling options either. If Apple are doing a hardware check then that is understandable.

You try using TinkerTool which is a third party tweaking tool that allows you to change many hidden settings including system font size.
https://www.bresink.com/osx/TinkerTool.html
